### PR TITLE
Convert gallery to C#6.0

### PR DESCRIPTION
**Pull request by maartenba**
This PR fixes:
* Convert gallery to C#6.0 #2815

### CODE REVIEW

**Comment by maartenba** [comment]
@xavierdecoster @joyhui Can you guys have a quick look? Basically just syntax updates.

File: src/NuGetGallery.Operations/Tasks/CreateWarehouseReportsTask.cs [hunk 1]
```diff
@@ -207,7 +207,7 @@ private void CreateDirtyPerPackageReports()
                 });
             });
 
-            string msg = string.Format("CreateDirtyPerPackageReports complete {0} seconds", (DateTime.Now - before).TotalSeconds);
+            string msg = $"CreateDirtyPerPackageReports complete {(DateTime.Now - before).TotalSeconds} seconds";
```
Comment (xavierdecoster) [inline src/NuGetGallery.Operations/Tasks/CreateWarehouseReportsTask.cs]: Whilst touching this code, should we prefer using `DateTime.UtcNow`? (multiple occurrences in this class)

Comment (maartenba) [inline src/NuGetGallery.Operations/Tasks/CreateWarehouseReportsTask.cs]: @joyhui What do you think? Sounds like a good idea to me.

**Comment by xavierdecoster** [comment]
Changes look good to me, did not hit any issues running the branch locally. Minor comment about usage of `DateTime.Now` instead of `DateTime.UtcNow`, but that's perhaps outside the scope of this PR, and can be addressed separately.

:shipit:

File: src/NuGetGallery.Operations/Tasks/CreateWarehouseReportsTask.cs (hunk 1 above)
Comment (maartenba) [inline src/NuGetGallery.Operations/Tasks/CreateWarehouseReportsTask.cs]: https://github.com/NuGet/NuGetGallery/issues/2860

**Pull request merged**